### ISSUE TITLE
Add a link to feature review at approval time

### ISSUE THREAD

**Issue by bliof** [FundingCircle/shipment_tracker#203]
When you click the details link from github on a merged PR you'll get to the feature review page without any "time" param specified.

There is a large text: " Feature Status: Approved at 2017-01-30 08:12:01 UTC", and we can make it a link so can go to the feature review at approval time.

**Comment by 0mega** [comment]
Yes that's useful, I think there is another time related improvement. In the Releases page if you click on the "Approved" link it will take you to the FeaturReview at the time of approval of the FeatureReview.
![screen shot 2017-02-03 at 15 20 54](https://cloud.githubusercontent.com/assets/519615/22596748/aa440402-ea24-11e6-98c4-f9c1f133648f.png)

But instead the time you see in the FeatureReview should be the time of the deploy event
